library/ssl_tls12_server.c

```diff
@@ -37,6 +37,13 @@
 #include "constant_time_internal.h"
 #include "mbedtls/constant_time.h"
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+#include "psa/crypto.h"
```

Comment:
This is already `#include`d by `ssl_misc.h` at line 33

Reply:
Thanks for pointing this out.